### DISCUSSION TITLE
Broken Links

### DISCUSSION THREAD

**Discussion by srynobio** [geneontology/helpdesk#384] [General]
In the past you hosted the following file via FTP.

ftp://ftp.geneontology.org/pub/go/doc/GO.xrf_abbs

But it looks like this ftp is down or retired. Could you update me on it status, and it's new location.

**Comment by kltm** [comment]
@srynobio 

For the most up-to-date version of this file, you should probably take a look at:
http://current.geneontology.org/metadata/
http://current.geneontology.org/metadata/GO.xrf_abbs

You may also want to consider the file that GO.xrf_abbs is derived from: http://current.geneontology.org/metadata/db-xrefs.yaml

**Comment by kltm** [comment]
For the status of FTP, after a long deprecation period, it was decommissioned a little over a year ago:
https://github.com/geneontology/go-announcements/issues/202
